### DISCUSSION TITLE
Login Problems

### DISCUSSION THREAD

**Discussion by zhlyyfvfv** [community/community#19184] [Copilot Conversations]
Failed to switch to the login page.
The plug-in shows that the first certificate is not connected to the network @bruce @tpope @arfon @davecheney @mattt 

---China

**Comment by davecheney** [comment]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?

**Reply by AEidSPL** [reply]
How to know the domain so I can ask for permission for it ?

**Reply by zhlyyfvfv** [reply]
I don't know.It used to work       This is an error message：[ERROR] [default] [2022-06-24T08:11:40.355Z] GitHub Copilot could not connect to server. Extension activation failed: "unable to verify the first certificate"

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or government, may be intercepting your SSL traffic?
